test/src/unit-capi-dense_array.cc

```diff
@@ -442,7 +442,7 @@ int* DenseArrayFx::read_dense_array_2D(
   // Error code
   int rc;
 
-  // Initialize a subarray
+  // Initialize a set_subarray
```

Comment:
A minor thing, but these look like overzealous find-replace modifications.

Reply:
Yes, it happened before. We should be careful when replacing keywords that may appear in text. Thanks for pointing this out. I will fix it.